### PR TITLE
DF-1347_Adding_Pagination_Validation

### PR DESCRIPTION
**Pull request by sebworks**
- Add better pagination validation

## Changes
- Updated _includes/post-macros.html to add notification class. 
- Added ' _includes/post-macros.html' to add notification logic.

## Test
- Navigate to 'http://localhost:7000/blog ' using Safari.
- Enter 't' in the pagination input field and click 'Go'
- You should see the notification message.

## Notes 
- @jimmynotjim , not sure if inserting container was best solution but was only way
to get notifier inserted.
- Updated markup to add notification below pagination form.

## Review 
- @jimmynotjim  
- @anselmbradford 
- @Scotchester  
- @KimberlyMunoz 

## Screenshots

Mobile:
screen shot 2015-05-22 at 12 14 48 pm

Tablet:
screen shot 2015-05-22 at 12 15 10 pm

Desktop:
screen shot 2015-05-22 at 12 15 30 pm

### CODE REVIEW

File: src/static/js/modules/pagination-validation.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * Set up DOM references and event handlers.
+ */
+function init() {
+  $( '.js-validate_pagination' ).each( function() {
+    var $form = $( this );
+    var $placeHolder = $( '<div class=\'cf-notification__pagination-ctr\'></div>' );
+    $form.parents( 'nav.pagination' ).before( $placeHolder );
+    $( $placeHolder ).cf_notifier();
```
Comment (jimmynotjim) [inline src/static/js/modules/pagination-validation.js]: You can attach `cf_notifier` to the form, no need for a placeholder. Then on submit, trigger the `cf_notifier:notify` event with your message in the options (if it's different from your default message).

Comment (jimmynotjim) [inline src/static/js/modules/pagination-validation.js]: On second thought, why not move this all into `form-validation.js`? I set that up to watch all forms, so you could easily extend it to send pagination messages as well.

**Comment by sebworks** [comment]
So if I attach it to the form then it screws up the layout of the pagination hence I created the placeholder. I might be able to attach it to the form and position it absolutely though.

File: src/static/js/modules/pagination-validation.js (hunk 1 above)
Comment (sebworks) [inline src/static/js/modules/pagination-validation.js]: ok, I'll add this into form-validation.js

**Comment by sebworks** [comment]
@jimmynotjim Unfortunately the placeholder is proving to be necessary. Once the insertion PR gets merged I'll create a new PR to add it into form-validation.js.

**Comment by jimmynotjim** [comment]
Sounds good. Tested and working well locally.

Side note, is the distance between the notification and the pagination set in a mockup? It seems a bit large breaking the relationship between the two. Might be a question for @ajbush or @duelj. Might be worth discussing in Team Review today.

**Comment by sebworks** [comment]
@jimmynotjim , updated the code and screenshots using 30px as the margin bottom.

**Comment by sebworks** [comment]
@jimmynotjim , updated the code and the notification below the pagination form.

**Comment by jimmynotjim** [comment]
:+1: from me :D

**Pull request merged**